recommend/report_text.go

```diff
@@ -63,13 +63,32 @@ func (r TextReport) SectionEnd(img *ImageInfo) error {
 }
 
 // Record addition of new text table row
-func (r TextReport) Record(ms MatchSpec, policyName string) error {
+func (r TextReport) Record(ms *MatchSpec, policyName string) error {
 	var rec []string
+	var severity, action string
+	if ms.Spec.File.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.File.Severity)
+	} else if ms.Spec.Process.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.Process.Severity)
+	} else if ms.Spec.Network.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.Network.Severity)
+	} else {
+		severity = fmt.Sprintf("%d", ms.Spec.Severity)
+	}
+	if ms.Spec.File.Action != "" {
+		action = string(ms.Spec.File.Action)
+	} else if ms.Spec.Process.Action != "" {
+		action = string(ms.Spec.Process.Action)
+	} else if ms.Spec.Network.Action != "" {
+		action = string(ms.Spec.Network.Action)
+	} else {
+		action = string(ms.Spec.Action)
```

Comment:
This code is duplicated. It could have been a common API that could be called from text and html report code.